tools/terraform_sync_tool/state.json

```diff
@@ -0,0 +1,6 @@
+{"@level":"info","@message":"Terraform 1.1.9","@module":"terraform.ui","@timestamp":"2022-06-08T13:22:12.564592-04:00","terraform":"1.1.9","type":"version","ui":"1.0"}
```

Comment:
Remove this file, no need to commit

Reply:
Output json file removed - https://github.com/candicehou07/bigquery-utils/commit/c2fc146d446acd55cb8965d0d520357c4e5760c3